github.go

```diff
@@ -100,7 +100,7 @@ func NewGitHubClient(source Source) (*GitHubClient, error) {
 
 func (g *GitHubClient) ListReleases() ([]*github.RepositoryRelease, error) {
 	opt := &github.RepositoryListByOrgOptions{
-		ListOptions: github.ListOptions{PerPage: 10},
+		ListOptions: github.ListOptions{PerPage: 100},
 	}
```

Comment:
In fixing up the test, I realized that we're not actually using this, so we could end up in an infinite loop!

```suggestion
	opt := &github.ListOptions{PerPage: 100}
```

And then change line 107:

```go
		releases, res, err := g.client.Repositories.ListReleases(context.TODO(), g.owner, g.repository, nil)
```

to

```go
		releases, res, err := g.client.Repositories.ListReleases(context.TODO(), g.owner, g.repository, opt)
```

Reply:
ah nice catch sorry for my negligence